### PR TITLE
ISAICP-5826 -Documentation added

### PR DESCRIPTION
**Pull request by Joosthe**

### CODE REVIEW

File: resources/mac/README.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+drush user:unblock
+````
+
+12 - create a new admin user 
+````
+drush uli
+````
+
+## Using the gulp
+For using the gulp please set your node version to v11.15.0 ( its recommanded to know your current node version so your can put ik back for newer projects ).
```
Comment (papegaill) [inline resources/mac/README.md]: For the moment, I will remove the Gulp part which should be part of the existing [FRONT-END documentation](https://github.com/ec-europa/joinup-dev/blob/develop/web/themes/joinup/README.md). 
An update should therefore be made on this part of the documentation.

There is also an open PR for updating Gulp and these dependencies: https://github.com/ec-europa/joinup-dev/pull/2005

File: README.md [hunk 2]
```diff
@@ -31,6 +31,9 @@ There are two ways to run Joinup. With `docker` and `docker-compose` and buildin
 
 To start with docker, please, check the separated [README file](resources/docker/README.md).
 
+### Mac without docker installation
+To start on a mac and want to do this without docker, please, check the separated [README file](resources/mac/README.md).
```
Comment (papegaill) [inline README.md]: ..To start on a mac without docker, please, check the separated..

Comment (Joosthe) [reply README.md]: here it says "...To start on a mac without docker, please, check the separated README file." with README file clickable

File: resources/mac/README.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+*“ Your site configuration active store is currently locked.”*  
+comment this line in **/web/sites/default/settings.php**  
+$settings['config_readonly'] = !file_exists(getcwd() . '/../disable-config-readonly');
+
+11 - unblock the admin user
+````
+drush user:unblock
+````
+
+12 - create a new admin user
```
[the remark this answers is not in the document]
Comment (Joosthe) [reply resources/mac/README.md]: drush uli by default is going update the default admin-user: user1

File: resources/mac/README.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+```
+
+7 - Setup solar and check if its running
+```  
+$ ./vendor/bin/phing setup-apache-solr
+```
+[Check Virtuoso](http://localhost:8890/sparql)  
+[Check Solr](http://localhost:8983/solr/#/)
+
+8 - Download Databases
```
[the remark this answers is not in the document]
Comment (Joosthe) [reply resources/mac/README.md]: I do agree on adding production but it's downloading multiple databases or thats what the command describes.

File: resources/mac/README.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+````
+drush uli
+````
+
+## Using the gulp
+For using the gulp please set your node version to v11.15.0 ( its recommanded to know your current node version so your can put ik back for newer projects ).
+To do this check:  
+[How do down or upgrade your node version.](https://www.surrealcms.com/blog/how-to-upgrade-or-downgrade-nodejs-using-npm.html)
+
+thenuse the following commands:
```
[the remark this answers is not in the document]
Comment (Joosthe) [reply resources/mac/README.md]: oke

File: resources/mac/README.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+````
+ $ cd web/themes/joinup/prototype ;
+ $ npm install;    
+ ````
+ check gulp file for all the possible tasks
+
+
+## Switching between branches
+This is you need to when you switch a branch to keep your content up to date
```
[the remark this answers is not in the document]
Comment (Joosthe) [reply resources/mac/README.md]: oke

File: resources/mac/README.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+````
+$ composer install
+````
+
+3 - Run build dev 
+```
+$ ./vendor/bin/phing build-dev
+```
+
+4 - install and/or relink
```
[the remark this answers is not in the document]
Comment (Joosthe) [reply resources/mac/README.md]: oke

File: resources/mac/README.md [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+$ ./vendor/bin/phing virtuoso-start
+$ ./vendor/bin/phing setup-virtuoso-permissions
+```
+
+6 - Run install dev
+```  
+$ ./vendor/bin/phing install-dev
+```
+
+7 - Setup solar and check if its running
```
[the remark this answers is not in the document]
Comment (Joosthe) [reply resources/mac/README.md]: oke

File: resources/mac/README.md (hunk 1 above)
Comment (Joosthe) [reply resources/mac/README.md]: Oke

File: resources/mac/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (papegaill) [reply resources/mac/README.md]: ok

File: resources/mac/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (papegaill) [reply resources/mac/README.md]: ok thanks for info ;-)

File: README.md (hunk 2 above)
Comment (papegaill) [reply README.md]: my suggestion was to remove _**and want to do this**_ from the sentence.

**Comment by claudiu-cristea** [comment]
@Joosthe, @papegaill I've only improved a little the document coding standards, fixed some typos. Thank you for your contribution.

**Pull request merged**